yt_dlp/extractor/sbs.py

```diff
@@ -122,6 +122,14 @@ def _real_extract(self, url):
         if traverse_obj(media, ('partOfSeries', {dict})):
             media['epName'] = traverse_obj(media, ('title', {str}))
 
+        # Need to set different language for forced subs or else they have priority over full subs
+        fixed_subtitles = {}
+        for lang, subs in subtitles.items():
+            for sub in subs:
+                if sub['url'].lower().endswith('_fe.vtt'):
+                    lang += '-forced'
```

Comment:
Isn't this problematic if there are multiple forced subtitles?

Reply:
I'm not that knowledgeable on subtitles; why would there be multiple  forced subs and what would the differences be between them?

Reply:
You would usually only have a single English forced subtitle track, but if for whatever reason we extracted these two URLs:
```
https://example.com/subtitle/1/eng_fe.vtt
https://example.com/subtitle/1/eng_alt_fe.vtt
```
then that would cause us to have `eng-forced` and `eng-forced-forced` as the languages.

Reply:
Imo. we should just do
```suggestion
                    fixed_subtitles.setdefault(f'{lang}-forced', []).append(sub)
                    continue
```